src/Client/Styles/Core/_resume.scss

```diff
@@ -0,0 +1,7 @@
+@import '../vars.scss';
+
+.resume {
+    .resume-dialog {
```

Comment:
Give one line space above the inner definition for better readability
`.resume{
           (space)
                .resume-dialog{...`

Reply:
fixed!